utils/pregenerate.js

```diff
@@ -18,14 +18,23 @@ export default `;
 
 async function writeApi() {
   const componentFolder = path.resolve('./lib/');
-  const files = await globby('**/K*.vue', { cwd: componentFolder });
+  const simpleFiles = await globby('**/K*.vue', { cwd: componentFolder });
```

Comment:
these changes were to allow the auto-building / extraction of both named (simple) files and files with a hierarchical folder structure. I've no idea why you have that or why this is so different to the keen build system, but it was necessary to stop errors when building.